Tools/generate_board_info_manifest.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+"""
+Script to print out the board information manifest in JSON format like this:
+
+{
+    "include":
+        [
+            {
+                "target": "px4_fmu-v5x_default",
+                "board_id": "51"
+            },
+        ...
+        ]
+}
+"""
+
+import argparse
+import os
+import json
+import re
+from kconfiglib import Kconfig
+
+parser = argparse.ArgumentParser(description='Generate build targets')
+
+parser.add_argument('-v', '--verbose', dest='verbose', action='store_true',
+                    help='Verbose Output')
+parser.add_argument('-p', '--pretty', dest='pretty', action='store_true',
+                    help='Pretty output instead of a single line')
+args = parser.parse_args()
+
+## GLOBAL VARIABLES
+verbose = args.verbose
+build_configs = []
+excluded_manufacturers = ['atlflight']
+
+# Labels (board variants) we will exclude
+excluded_labels = [
+    'stackcheck', 'nolockstep', 'replay', 'test',
+    'uavcanv1' # TODO: fix and enable
+    ]
```

Comment:
Do you actually need to exclude any?

Reply:
I didn't want to include the targets that actually doesn't exist in the server as a binary (which is dependent on the script `generate_board_targets_json.py` file. Shall I rather exclude the 'excluded-label' filter on both scripts then?